App/src/main/resources/application.properties

```diff
@@ -9,3 +9,9 @@ spring.jpa.show_sql=true
 trust.store=classpath:keystore/managerkey.p12
 #trust store password
 trust.store.password=passwd
+
+#System Manager Urls
+systemManager.url.user=https://localhost:8081/api/user
+systemManager.url.group=https://localhost:8081/api/group
+systemManager.url.firewall=https://localhost:8081/api/firewall
+systemManager.url.directory=https://localhost:8081/api/dir
```

Comment:
sa mi trocha nepaci to opakovanie, radšej by som dal `systemManager.api.url=https://localhost:8081/api`, a tie /user atd vytiahnut niekam do konštant, asi bude stacit aj len v `SystemManagerService`, inde sa to pouzivat nebude, a skladat to až v appke teda